src/metabase/sync/sync_metadata/tables.clj

```diff
@@ -189,32 +209,32 @@
   ([database :- i/DatabaseInstance db-metadata]
    ;; determine what's changed between what info we have and what's in the DB
    (let [db-tables               (table-set db-metadata)
-         our-metadata            (our-metadata database)
-         strip-desc              (fn [metadata]
-                                   (set (map #(dissoc % :description) metadata)))
+         name+schema->db-table   (m/index-by (juxt :name :schema) db-tables)
+         our-metadata            (db->our-metadata database)
+         keep-table-id-info      (fn [metadata]
```

Comment:
This change somehow changes the order of sample dataset tables added on startup. 

that's why you'll see some tests affected by this, one of which is I have to update the table ides in `e2e/support/cypress_data.js`

Reply:
it's a function so I think the name should contains a verb

how about `keep-schema+name-set`?

Reply:
Function names don't need to contain verbs, but sure `keep-schema+name-set` is better too.